sql/analyzer/fix_exec_indexes.go

```diff
@@ -280,7 +281,21 @@ func (s *idxScope) visitSelf(n sql.Node) error {
 		for i, c := range n.Destination.Schema() {
 			rightSchema[i] = c
 			if _, ok := n.Source.(*plan.Values); !ok && len(n.Destination.Schema()) == len(n.Source.Schema()) {
-				rightSchema[len(n.Destination.Schema())+i] = n.Source.Schema()[i]
+				// find source index that aligns with dest column
+				var matched bool
+				for j, c3 := range n.ColumnNames {
```

Comment:
(minor) "c3" isn't a great variable name. It's not too hard to figure it out, but seems like a better var name could improve readability.

Reply:
fixed all of the naming in this func to be clearer